src/common.build.pre.props

```diff
@@ -77,7 +77,8 @@
   </ItemGroup>
 
   <PropertyGroup Label="Globals">
-    <WindowsTargetPlatformVersion Condition="'$(WindowsTargetPlatformVersion)' == ''">10.0.18362.0</WindowsTargetPlatformVersion>
+    <WindowsTargetPlatformVersion Condition="'$(WindowsTargetPlatformVersion)' == ''">10.0.19041.0</WindowsTargetPlatformVersion>
+    <WindowsTargetPlatformMinVersion Condition="'$(WindowsTargetPlatformMinVersion)' == ''">10.0.18362.0</WindowsTargetPlatformMinVersion>

```

Comment:
Is it valuable to continue expressing a minimum version of 10.0.17763? Technically we work down to 7.0 for some components, but we have not been good about making that boundary "clean"

Reply:
i guess 7.0 is "6.1" :smile:

conhost itself works down to 8.1, which is ... 6.2? Hell, our version number story is bad.

Reply:
Should we specify `WindowsTargetPlatformMinVersion` in the first place? It's optional as far as I understood it...

Reply:
It may help with some API availability checks/warnings. I'd be comfortable leaving it at 17763, so that we can establish a baseline for things like WinConPTY and conhost. :smile: